watertap/unit_models/tests/test_aeration_tank.py

```diff
@@ -103,7 +119,6 @@ def test_config():
     assert m.fs.unit.config.property_package is m.fs.properties
     assert m.fs.unit.config.reaction_package is m.fs.reactions
     assert m.fs.unit.config.electricity_consumption == ElectricityConsumption.calculated
-    assert m.fs.unit.config.has_aeration
```

Comment:
Why was this line deleted?

Reply:
Must've been deleted unintentionally somehow. I've added it back